ui/lib/pki/addon/routes/issuers/issuer/details.js

```diff
@@ -3,19 +3,23 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-import PkiIssuerIndexRoute from './index';
+import PkiIssuerRoute from '../issuer';
 import { verifyCertificates } from 'vault/utils/parse-pki-cert';
-export default class PkiIssuerDetailsRoute extends PkiIssuerIndexRoute {
-  // Details route gets issuer data from PkiIssuerIndexRoute
-  async setupController(controller, resolvedModel) {
+import { hash } from 'rsvp';
+export default class PkiIssuerDetailsRoute extends PkiIssuerRoute {
+  model() {
+    const issuer = this.modelFor('issuers.issuer');
+    return hash({
+      issuer,
+      pem: this.fetchCertByFormat(issuer.id, 'pem'),
+      der: this.fetchCertByFormat(issuer.id, 'der'),
+      isRotatable: this.isRoot(issuer),
+    });
+  }
+
+  setupController(controller, resolvedModel) {
     super.setupController(controller, resolvedModel);
-    controller.breadcrumbs.push({ label: resolvedModel.id });
-    const pem = await this.fetchCertByFormat(resolvedModel.id, 'pem');
-    const der = await this.fetchCertByFormat(resolvedModel.id, 'der');
-    const isRoot = await this.isRoot(resolvedModel);
```

Comment:
I moved this to the model hook because on refresh (not just hot reloading) the `isRoot` data wasn't consistently coming in! 

I don't think there was an issue with the pem/der data so I can put that back!